mopro-msm/src/msm/metal_msm/shader/cuzk/barrett_reduction.metal

```diff
@@ -4,6 +4,9 @@ using namespace metal;
 #include <metal_stdlib>
 #include <metal_math>
 #include "../field/ff.metal"
+#include <metal_logging>
+constant os_log logger(/*subsystem=*/"barret_reduction", /*category=*/"abc");
```

Comment:
It's here @moven0831 (the place where subsystem and category defined), the pretty same definition exists in the next updated shader.